api/authentication.thrift

```diff
@@ -28,4 +29,10 @@ service codeCheckerAuthentication {
   // performs logout action for the user (must be called from the corresponding valid session)
   bool destroySession()
              throws (1: shared.RequestFailed requestError)
```

Comment:
Though it seems Thrift doesn't seem to complain about this, just for the sake of code appearance, put a `,` at the end of this function declaration.